index.js

```diff
@@ -0,0 +1,25 @@
+import { DateTime } from '../../node_modules/luxon/src/luxon.js';
```

Comment:
Great design @binodbhusal 🤝

- [ ] As stated in the project requirements, your design is missing the time which is supposed to be displayed below the navigation bar. I would recommend removing the `Luxon.js` file from the node_modules directory and add it to the `modules` directory and then import it from there.
![no time](https://user-images.githubusercontent.com/63917725/235552512-d4715a30-e1fc-4e22-8eb4-6f747cf6c5f7.PNG)

Reply:
- [ ] Good work implementing this too 👍🏾